source/tests/system/nidigital_driver_api_tests.cpp

```diff
@@ -79,7 +113,23 @@ class NiDigitalDriverApiTest : public ::testing::Test {
 
     ::grpc::Status status = GetStub()->Close(&context, request, &response);
 
-    EXPECT_TRUE(status.ok());
+    EXPECT_SUCCESS(status);
+    expect_api_success(response.status());
+  }
+
+  void close_multi_instrument_driver_session()
+  {
+    if (!multi_instrument_driver_session_) {
```

Comment:
Can this case actually happen?

Reply:
I'm sure it's just as likely as the equivalent check in `close_driver_session()`, from which I copied this function. Can it actually happen, though? Probably, yeah, if the initialization functions fail.